src/routes/notifications/notifications.service.ts

```diff
@@ -76,6 +76,27 @@ export class NotificationsService {
     }
   }
 
+  /**
+   * Sends a delete request for a chain, notification uuid and safe address.
```

Comment:
Not sure if I completely get this sentence 😅 Is it enumerating the parameters?

Reply:
Yes, that was the intention but it's poorly written and redundant also 🙂 I will remove it.

Reply:
Fixed in [d880fcb](https://github.com/5afe/safe-client-gateway-nest/pull/268/commits/d880fcb3732f5bd60894507bcb3104e4a46b7714)

Reply:
Rephrased in [77a2868](https://github.com/5afe/safe-client-gateway-nest/pull/268/commits/77a2868a5ebb4ce4774cb16efc978618b404e525)